src/modules/layoutValidation.js

```diff
@@ -68,3 +68,16 @@ export const validateLayout = (layout) => {
             return validateLineListLayout(layout)
     }
 }
+
+export const layoutHasProgramId = (layout) => {
+    if (!layout) {
+        return false
+    }
+    switch (layout.type) {
+        case VIS_TYPE_LINE_LIST:
+        default:
+            return Boolean(layout?.program?.id)
+    }
+}
+
+export const AOIsLegacy = (layout) => layout.legacy // The legacy prop is returned by the backend for AOs that were created in the old ER app
```

Comment:
bikeshedding here. Could the function be renamed to `aoCreatedInEventReportsApp`?

Reply:
renamed ✔️